.github/workflows/run_test.yml

```diff
@@ -144,6 +144,13 @@ jobs:
                   command: 'npm install'
                   working-directory: ${{ matrix.test.path }}
 
+            - name: Install experimental dependencies for experimental tests
+              if: matrix.azle_source == 'link' && ${{ inputs.run-experimental }}
+              uses: ./.github/actions/retry_command
+              with:
+                  command: 'npm install github:demergent-labs/azle-experimental-deps#34cf2e34958e03cae7bf63f77f86ae682eef8a7c'
```

Comment:
Let's use the https URL in our npm install commands

Reply:
done